docs/notebooks/entropic_map_estimator.ipynb

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.    plt.scatter(x[0:150,0],x[0:150,1],c='r',marker='.',label='Source')
</pre><p>same plot function should be used</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>